forc/tests/Forc.toml

```diff
@@ -0,0 +1,8 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "tests"
```

Comment:
This sample project should be under `forc/tests/<project_name>`

It should also have a more meaningful name, since all the projects under `tests` are used for testing. 

The binary file should probably not be checked in.